common/planners/src/plan_expression.rs

```diff
@@ -90,6 +90,11 @@ impl Expression {
     pub fn column_name(&self) -> String {
         match self {
             Expression::Alias(name, _expr) => name.clone(),
+            Expression::ScalarFunction { op, args: _ } => match op.as_ref() {
```

Comment:
Better to use `Expression::ScalarFunction {op, ..}` ?

Reply:
ok